lib/ceph-ruby/cluster.rb

```diff
@@ -2,15 +2,18 @@ module CephRuby
   class Cluster
     attr_accessor :handle
 
-    def initialize(config_path = "/etc/ceph/ceph.conf")
+    def initialize(options = {})
+      options[:config_path] ||= "/etc/ceph/ceph.conf"
+      options[:username] ||= nil
```

Comment:
No need to initialize this key here - when we later access `options[:username]` and it's not defined it'll return nil anyway.